ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/InterfaceSubscriptionsIT.scala

```diff
@@ -70,9 +76,10 @@ class InterfaceSubscriptionsIT extends LedgerTestSuite {
   })
 
   test(
-    "ISTransactionsCreateArgumentsBlob",
+    s"${prefix}TransactionsCreateArgumentsBlob",
```

Comment:
Wouldn't this test be run twice with same setup?

Reply:
Ha! Great you are being so attentive!
But not, the `interfaceFilter` function will adopt itself to either old or new blobs depending on the test suite

Reply:
ah yeah `useTemplateIdBasedLegacyFormat` is in class scope and you needn't pass it here 👍